xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/SaveAction.java

```diff
@@ -402,11 +402,11 @@ private boolean isConflictingWithVersion(XWikiContext context, XWikiDocument ori
         Version previousVersion = new Version(request.getParameter("previousVersion"));
         Version latestVersion = originalDoc.getRCSVersion();
 
-        DateTime editingVersionDate = new DateTime(request.getParameter("editingVersionDate"));
-        DateTime latestVersionDate = new DateTime(originalDoc.getDate());
+        Date editingVersionDate = new Date(Long.parseLong(request.getParameter("editingVersionDate")));
```

Comment:
Does this really makes a difference or was it just to remove a dependency on joda (which perfectly make sense too) ? It's not obvious to me when looking at the diff.

Reply:
It wasn't just to remove the dependency. The main idea was to be able to easily send a date from Java or from Javascript without having to mind about the exact format. I was using `editingVersionDate.toISOString()` in Javascript before, and when I wrote the velocity code I didn't find immediately the exact same API for it. So all in all I decided to rely on a solution that I found a bit more standard.

Reply:
This can be considered a breaking change and should be documented in the release notes. We need to update at least [this place](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-edit/xwiki-platform-edit-ui/src/main/resources/XWiki/InplaceEditing.xml#L820).

Reply:
Thanks for the pointer. It's a breaking change but I'm not sure it has ever been documented, so it would need to be documented in RN to be safe but the impact should be limited. Or do you think we should keep it like that?

Reply:
So this is indeed a breaking change of what is usually seen as an API (standard action parameters). Sounds a bit dangerous but I don't know how old and used this is.

Reply:
It's relatively recent and I doubt it's used outside platform. I'm fine to change the format (I prefer the timestamp over the ISO format), as long as we update the in-place editing code.

Reply:
>  as long as we update the in-place editing code.

Note that I did edited it: https://github.com/xwiki/xwiki-platform/pull/1409/files#diff-ff70cc3aca6d799932e43f371b498c5eR820